meetings/2023-05/may-15.md

```diff
@@ -0,0 +1,800 @@
+
+### Summary of Key Points
+* Implementers found this edge case where the constructor is called twice, and we discussed improvement of making that happen only once.
+
+### Conclusion/Decision
+* The committee agreed to consensus on PR #41 to fix the double construction issue
+
+## Well-Formed Unicode Strings for Stage 4
+Presenter: Micheal Ficarra (MF)
+
+- [proposal](https://github.com/tc39/proposal-is-usv-string)
+- [slides](https://docs.google.com/presentation/d/1_y2n2wK2NIDtUeHZZcb3mxEezRMJ7m_7QBQO50NhxbA/edit#slide=id.p)
+- [PR](https://github.com/tc39/ecma262/pull/3039)
+
+MF: This is the entire text for the well-formed strings proposal. The proposal adds isWellFormed and toWellFormed. We did not iterate on this too much since this was uncontroversial through its whole life.
+
+MF: As far as implementation status we have two shipping, Safari and Chrome since March,
+Firefox has recently implemented this proposal, and we have polyfills in core-js and also these methods are polyfilled individually.
+
+MF: We have tests from JHD, he wrote tests a while back and they were merged a while back. And we have a PR for 262 that has been approved by the other editors. So in summary, we have I believe met all criteria for stage 4. And that is all for stage 4.
+
+KG: I support this for stage 4.
+
+USA: Thank you KG, and we have DLM and CDA supporting stage 4. So it looks like you have quite a bit of support. So there is no opposing comments? Not so far. All right, I believe you have consensus. Congratulations, MF for stage 4. We have champagne I believe.
+
+MF: Thank you.
+
+### Conclusion/Decision
+* Consensus for stage 4 for Well-Formed Unicode Strings
+
+## Atomics.waitAsync for Stage 4
+
+Presenter: Shu-yu Go (SYG)
+
+- [proposal](https://github.com/tc39/proposal-atomics-wait-async)
+- [PR](https://github.com/tc39/ecma262/pull/3049)
+
+SYG: Let me zoom in a bit. So, this is async news of atomics.waitAsync. And this is for blocking the thread, and this is originally motivated because you cannot block the main thread on the environment browser, and that is policy. And so if you want to interact with worker threads that use `Atomics.notify` notify and only option is to use async version, and this is quite old, and we have not presented this for a while, and Chrome got stage two or three years ago and the thread in’ 87 and some time in November of 2020. And then it stopped for a while, and so Safari shipped in 16.4. So this PR is the spec text PR against the main line spec text. And it has gotten fair bunch of reviews at this point and it rating and it is good to go. And there is an HTML component to this because it needs a bunch of post interaction to in queue task to resolve promises across threads or as we call them agents and there are 60 tests. So with that, I would like to and for stage 4 for `Atomics.waitAsync.
```

Comment:
Missing backtick is breaking the syntax highlighting for the rest of the file.
```suggestion
SYG: Let me zoom in a bit. So, this is async news of atomics.waitAsync. And this is for blocking the thread, and this is originally motivated because you cannot block the main thread on the environment browser, and that is policy. And so if you want to interact with worker threads that use `Atomics.notify` notify and only option is to use async version, and this is quite old, and we have not presented this for a while, and Chrome got stage two or three years ago and the thread in’ 87 and some time in November of 2020. And then it stopped for a while, and so Safari shipped in 16.4. So this PR is the spec text PR against the main line spec text. And it has gotten fair bunch of reviews at this point and it rating and it is good to go. And there is an HTML component to this because it needs a bunch of post interaction to in queue task to resolve promises across threads or as we call them agents and there are 60 tests. So with that, I would like to and for stage 4 for `Atomics.waitAsync`.
```